doc/design/scope.md

```diff
@@ -107,11 +107,11 @@ class Scope {
 ```
 ## Only scope can create a variable
 
-To ensure `only scope can create a variable`, we should mark `Variable`'s constructor as a private member function, and Scope is a friend class of Variable. And then only `CreateVariable` can construct `Variable`.
+To ensure `only scope can create a variable`, we should mark `Variable`'s constructor as a private member function, and Scope is a friend class of Variable. And then only `NewVar` can construct `Variable`.
```

Comment:
no only `Variable`, `Operator`, `Tensor`, `Scope` are all `DISALLOW_COPY_AND_ASSIGN`, this macro will make default and copy constructor `= delete`.